integration/airflow/openlineage/lineage_backend/__init__.py

```diff
@@ -141,3 +139,15 @@ def _get_location(task):
     @staticmethod
     def _now_ms():
         return int(round(time.time() * 1000))
+
+
+class OpenLineageBackend(LineageBackend):
+    # Airflow 1.10 uses send_lineage as staticmethod, so just construct class
+    # instance on first use and delegate calls to it
```

Comment:
How does Airflow 2.0 work with this? Will it also call this as a static method? Or do users need to start configuring lineage with `openlineage.lineage_backend.Backend`?

Reply:
There's no difference in Airflow 2.
In Python, you can call static methods as class methods or as instance methods:

```
Python 3.9.2 (default, May  5 2021, 14:01:15) 
[GCC 10.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> class A:
...     @staticmethod
...     def a():
...             return 2
... 
>>> A.a()
2
>>> A().a()
2
```